projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.7-a.5 - 2022-02-16
+### Enhancements
+- Form block: add support for color and spacing block options.
+- Mailchimp block: add align, color and spacing features
+- Podcast Player block: add Align (Wide and Full Width) and Spacing (Padding and Margin) tools
+- Premium Content Block: add a "Remove block and keep content" settings menu item and fixed transformations to groups.
+- Premium Content Block: don't allow core markup blocks to be transformed to Premium Content block (e.g. core/nextpage, core/spacer).
+- Premium Content Block: when content within a premium-content view is selected in the editor, switch to that view.
+- Related Posts block: add design tools (align, color, typography & spacing).
+- Repeat Visitor block: Add the Align tool (Wide and Full Width).
+- Dashboard: update the Free plan description to consider the currently active products.
```

Comment:
Since the Premium Content block is only available on WordPress.com, I would move all this to other.

I would also suggest capitalizing "Block" everywhere.